vignettes/details.Rmd

```diff
@@ -55,7 +55,7 @@ Environment variables for a task are computed by collecting (in increasing order
 
 * `hipercow` default environment variables (defined in hipercow's `DEFAULT_ENVVARS`); currently `r inline(paste(apply(hipercow:::DEFAULT_ENVVARS[c("name", "value")], 1, paste, collapse = "="), collapse = ", ")
 )`
-* driver-specific variables (defined by the `default_envvars` argument to `hipercow_driver`); see `vignette("windows")` for the windows defaults
+* driver-specific variables (defined by the `default_envvars` argument to `hipercow_driver`); see `vignette("dide-cluster")` for defaults specific to our DIED cluster
```

Comment:
```suggestion
* driver-specific variables (defined by the `default_envvars` argument to `hipercow_driver`); see `vignette("dide-cluster")` for defaults specific to our DIDE cluster
```

I managed to make the same typo when trying to fix this, obviously it really wants death